sdks/python/apache_beam/runners/direct/evaluation_context.py

```diff
@@ -224,7 +225,10 @@ def handle_result(
                 self._side_inputs_container.finalize_value_and_get_tasks(view))
 
       if result.counters:
+        # there is no merge fn in a distribution counter
```

Comment:
Shouldn't there be?

Reply:
For current purpose, we only update counter and translate counter into update_metric_proto. How about add 'pass' to distribution_counter.merge func?

Reply:
What I don't understand is why we have a special case to skip it in this case (and not others). 

If we want a dummy implementation, raising NotImplemntedError is preferable to a pass in case someone tries to call it.

Reply:
Right, I've made some changes in latest revision. Actually, I'll translate DistributionMetadataAccumulator into DistributionAccumulator in batchworker. So for a counter_factory in one work_item, DistributionMetadataAccumulator is the only one new counter I added. I'll raise NotImplemntedError in next commit. Thanks!